src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1686,9 +1686,10 @@ private int InternalVerticalScrollPosition
         }
 
         /// <summary>
+        /// PrimaryCursorPosition property.<br />
         /// Specify primary cursor (caret) position in text control.
```

Comment:
Please consider changing the order to avoid confusion about which part is "primary" - at least I was confused for a while. Description of PrimaryCursorColor at line 367 is better.
```suggestion
        /// Specify the position of the primary cursor (caret) in text control.
```

Reply:
@wiertel ,
The suggestion has been applied. Could you check it , please.